Sources/Vortex/Helpers/Equatable+Identifiable.swift

```diff
@@ -0,0 +1,14 @@
+//
+// Equatable+Identifiable.swift
+// Vortex
+// https://www.github.com/twostraws/Vortex
+// See LICENSE for license information.
+//
+
+
+// Automatic conformance to Equatable for any Identifiable Type.
+extension Equatable where Self: Identifiable{
```

Comment:
I don't think this is a good idea – `Equatable` and `Identifiable` have different meanings, and conflating the two seems risky in the long term. The same is true of the `Hashable` extension. If this is just to avoid the two `static` methods in `ParticleSystem`, I think I'd rather keep the methods. Is there something I'm missing?